src/libmeasurement_kit/ooni/constants.hpp

```diff
@@ -56,6 +56,8 @@ const http::Headers COMMON_CLIENT_HEADERS = {
   }
 };
 
+const std::string MEEK_SERVER_RESPONSE = "I\u2019m just a happy little web server.\n";
```

Comment:
I don't know. I guess, to reply the first thing I need to do is to understand what unicode literals do to a C string, because honestly I don't know that :-).